ql/qldefines.hpp

```diff
@@ -187,11 +187,15 @@
 // emit warning when using deprecated features
 #if defined(BOOST_MSVC)       // Microsoft Visual C++
 #define QL_DEPRECATED __declspec(deprecated)
+#define QL_FORCE_NONINLINE __declspec(noinline)
```

Comment:
I would move this to `version.cpp`, to avoid triggering a recompilation of the whole library for people using the latest version.